README.md

```diff
@@ -1 +1,3 @@
-# MtnMentalHealth
```

Comment:
Why did you delete this and put it back?

Reply:
Because I was learning the system.